### ISSUE TITLE
Deprecation warning PHP7.4

### ISSUE THREAD

**Issue by abishekrsrikaanth** [mikeerickson/laravel-craftsman#12]
When using this package with PHP 7.4, get the following errors

```
PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/codedungeon/laravel-craftsman/app/helpers.php on line 43

Deprecated: Array and string offset access syntax with curly braces is deprecated in vendor/codedungeon/laravel-craftsman/app/helpers.php on line 43

PHP Deprecated:  Array and string offset access syntax with curly braces is deprecated in vendor/codedungeon/laravel-craftsman/app/helpers.php on line 49

Deprecated: Array and string offset access syntax with curly braces is deprecated in vendor/codedungeon/laravel-craftsman/app/helpers.php on line 49
```

**Comment by mikeerickson** [comment]
I am working on a major refactor to address these issues. The bigger fix is to migrate to Laravel-Zero 6.x

**Comment by mikeerickson** [comment]
Fixed in [1.12](https://github.com/mikeerickson/laravel-craftsman/releases/tag/1.12.0)
